spdx/config.py

```diff
@@ -32,7 +33,7 @@ def __delitem__(self, key):
 
 def load_license_list():
     FILE_NAME = os.path.join(os.path.dirname(__file__), 'spdx_licenselist.csv')
-    with open(FILE_NAME, 'r') as file_in:
+    with codecs.open(FILE_NAME, 'r', encoding="utf-8") as file_in:
```

Comment:
I know it is not your doing, but improving the code while you touch it is a good idea.  FILE_NAME and uppercase names are used for global constants by convention and not scoped variables. Use `file_name` instead.